app/views/groups/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<p style="color: green"><%= notice %></p>
+
+<div class='bg-white pt-2'>
+  <%= render @group %>
+</div>
+
+<div class='d-flex justify-content-center gap-3 mb-2'>
+  <%= link_to "Edit this group", edit_group_path(@group) %> |
+  <%= link_to "Delete this category", @group, data: { turbo_method: :delete, turbo_confirm: "Are you sure?" } %>
+</div>
+
+<div class='px-2 '>
+
+  <% @group.movements.order('created_at DESC').each do |movement| %>
+
+    <%= link_to group_movement_path(@group,movement) do %>
+
+      <div class='link-dark'>
+        <%= render movement %>
+      </div>
+      
+    <% end%>
+    <%# <%= link_to "Show this movement", group_movement_path(@group,movement) %>
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
@BenMukebo You are absolutely right, thank you very much for your comments. I have already applied your comments and applied your advice.

Reply:
This change has been done well